dd-sdk-android-ndk/src/main/kotlin/com/datadog/android/ndk/NdkCrashReportsPlugin.kt

```diff
@@ -61,6 +63,15 @@ class NdkCrashReportsPlugin : DatadogPlugin {
         unregisterSignalHandler()
     }
 
+    override fun onContextChanged(context: DatadogContext) {
+        // TODO: RUMM-637 Only update the rum context if the `bundleWithRum` config attribute is true
+        context.rum?.let {
+            updateRumContext(it.applicationId, it.sessionId, it.viewId)
```

Comment:
On the naming side, I'd rather have something more explicit than `bundleWithRum`, like `RumUpdatesEnabled`. Anyway one can also keep the previous context and check if the `newContext.rum != oldContext.rum` ;)
But we can do that in a further ticket

Reply:
That was my intention but for now I don't think this is needed. I am going to add this in a further ticket once I find a suitable way to unit test this class.